integration_tests/tests/in_cluster.rs

```diff
@@ -47,103 +48,163 @@ impl InClusterJanusPair {
+    ///  - `JANUS_E2E_DIVVIUP_API_NAMESPACE`: The Kubernetes namespace where `divviup-api` is
+    ///     deployed.
     async fn new(vdaf: VdafInstance, query_type: QueryType) -> Self {
-        let (collector_private_key, mut leader_task, mut helper_task) =
-            test_task_builders(vdaf, query_type);
-        leader_task = leader_task.with_min_batch_size(100);
-        helper_task = helper_task.with_min_batch_size(100);
-
         let (
             kubeconfig_path,
             kubectl_context_name,
             leader_namespace,
-            leader_api_namespace,
+            leader_aggregator_api_auth_token,
             helper_namespace,
-            helper_api_namespace,
+            helper_aggregator_api_auth_token,
+            divviup_api_namespace,
         ) = match (
             env::var("JANUS_E2E_KUBE_CONFIG_PATH"),
             env::var("JANUS_E2E_KUBECTL_CONTEXT_NAME"),
             env::var("JANUS_E2E_LEADER_NAMESPACE"),
-            env::var("JANUS_E2E_LEADER_API_NAMESPACE"),
+            env::var("JANUS_E2E_LEADER_AGGREGATOR_API_AUTH_TOKEN"),
             env::var("JANUS_E2E_HELPER_NAMESPACE"),
-            env::var("JANUS_E2E_HELPER_API_NAMESPACE"),
+            env::var("JANUS_E2E_HELPER_AGGREGATOR_API_AUTH_TOKEN"),
+            env::var("JANUS_E2E_DIVVIUP_API_NAMESPACE"),
         ) {
             (
                 Ok(kubeconfig_path),
                 Ok(kubectl_context_name),
                 Ok(leader_namespace),
-                Ok(leader_api_namespace),
+                Ok(leader_aggregator_api_auth_token),
                 Ok(helper_namespace),
-                Ok(helper_api_namespace),
+                Ok(helper_aggregator_api_auth_token),
+                Ok(divviup_api_namespace),
             ) => (
                 kubeconfig_path,
                 kubectl_context_name,
                 leader_namespace,
-                leader_api_namespace,
+                leader_aggregator_api_auth_token,
                 helper_namespace,
-                helper_api_namespace,
+                helper_aggregator_api_auth_token,
+                divviup_api_namespace,
             ),
             _ => panic!("missing or invalid environment variables"),
         };
 
         let cluster = Cluster::new(&kubeconfig_path, &kubectl_context_name);
 
+        let (collector_private_key, task_builder, _) = test_task_builders(vdaf, query_type);
+        let task = task_builder.with_min_batch_size(100).build();
+
         // From outside the cluster, the aggregators are reached at a dynamically allocated port on
         // localhost. When the aggregators talk to each other, they do so in the cluster's network,
         // so they need the in-cluster DNS name of the other aggregator, and they can use well-known
         // service port numbers. The leader uses its view of its own endpoint URL to construct
         // collection job URIs, so we will only patch each aggregator's view of its peer's endpoint.
```

Comment:
Since we no longer directly construct new task request bodies from `Task` objects, and instead send in-cluster URLs when setting up each aggregator in divviup-api, the end of this comment isn't applicable anymore.
```suggestion
        // From outside the cluster, the aggregators are reached at a dynamically allocated port on
        // localhost. When the aggregators talk to each other, they do so in the cluster's network,
        // so they need the in-cluster DNS name of the other aggregator, and they can use well-known
        // service port numbers.
```